userguide/jails.rst

```diff
@@ -4,41 +4,40 @@
 Jails
 =====
 
-Jails are a very light-weight, operating system-level virtualization,
-and the basis for the %brand% :ref:`plugin system <Plugins>`. Jails
-can also be created by the user. While plugins generally are
-preconfigured and use a GUI for configuration, jails are often managed
-from the command line. Plugins provide a single application or
-service, while jails can provide multiple services.
-
-By default, a
-`FreeBSD jail <https://en.wikipedia.org/wiki/Freebsd_jail>`__
-is created. A jail is an independent instance of FreeBSD, using the host
-kernel, running on the same hardware without all of the overhead
-usually associated with virtualization. The jail installs FreeBSD
-software management utilities so FreeBSD ports can be compiled and
-FreeBSD packages can be installed from the command line of the jail.
+
+Jails are a lightweight, operating-system-level virtualization.
+Jails are also used as the basis for the %brand%
+:ref:`plugin system <Plugins>`. Jails are created by the user and can
+provide multiple services, while plugins are preconfigured for a single
+application.
+
+`FreeBSD jails <https://en.wikipedia.org/wiki/Freebsd_jail>`__
+are created bby default. These jails are independent instances of
+FreeBSD. The jail uses the host hardware and runs on the host kernel,
+avoiding most of the overhead usually associated with virtualization.
+The jail installs FreeBSD software management utilities so FreeBSD
+packages or ports can be installed from the jail command line. This
+allows for FreeBSD ports to be compiled and FreeBSD packages to be
+installed from the command line of the jail.
 
 It is important to understand that users, groups, installed software,
 and configurations within a jail are isolated from both the %brand%
 host operating system and any other jails running on that system.
-During creation, the :guilabel:`VNET` option can be checked to provide
-the jail with an independent networking stack. The jail can then do its
-own IP broadcasting, which is required by some applications.
+
+During creation, set the :guilabel:`VNET` option to provide
+the jail with an independent networking stack. The jail is then able to
+broadcast an IP address, which is required by some applications.
 
 Advanced users can also create custom templates to automate the
 creation of pre-installed and customized operating systems.
 
-The ability to create multiple jails offers great flexibility
+The ability to create multiple jails offers flexibility
 regarding software management. For example, an administrator can
 choose to provide application separation by installing different
 applications in each jail, to create one jail for all installed
 applications, or to mix and match how software is installed into each
 jail.
 
```

Comment:
If we can find an appropriate link, let's add:

.. note:: Some distributions of Linux can run in a jail. See
   `Linux in a FreeBSD jail <...>`__.

Reply:
In progress.